### ISSUE TITLE
Clipboard use

### ISSUE THREAD

**Issue by alexreg** [defunkt/gist#217]
The utility seems to use `xclip` before `pbcopy` if both are present on the system. This is undesirable behaviour for me, on OS X. Can we configure it to use pbcopy first?

**Comment by ConradIrwin** [comment]
Sure, should be an easy pull request ;)

**Comment by alexreg** [comment]
I’m afraid I don’t know any Ruby! Also, is just reversing the order fine, or do you want to add an option for it? If so then it would definitely be beyond my abilities.

**Comment by ConradIrwin** [comment]
I don't think an option is needed. I was imagining that they were mutually exclusive — presumable you installed xclip with quartz on a mac?

**Comment by alexreg** [comment]
Indeed. XClip is still useful for some apps on Mac, but pbcopy/pbpaste is the one that should be used for command line utilities, really.

**Comment by alexreg** [comment]
I don't know how to build the utility, but I've submitted a pull-request for what I think is the fix.
